### DISCUSSION TITLE
CouchDB 3.1.1

### DISCUSSION THREAD

**Discussion by grueni** [OpenIndiana/oi-userland#6728] [General]
Hello

I would like to send a PR for CouchDB 3.1.1.
But the package couchdb-common is obsoleted.
Could this package be resurrected?

Thank you

**Comment by AndWac** [comment]
Am 08.05.21 um 11:11 schrieb grueni:
>
> Hello
>
> I would like to send a PR for CouchDB 3.1.1.
> But the package couchdb-common is obsoleted.
> Could this package be resurrected?
>
> Thank you
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/discussions/6728>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADET4N77YXR5XETCT7JKTADTMT53TANCNFSM44MQRYSA>.
>
I am not sure how to do that and what the implications will be.

couchdb-21, couchdb-common and service/database/couchdb-21 have been
obsoleted because nobody seemed to care for them.
What will be the contents of couchdb-common (what will be its contents)?

**Comment by grueni** [comment]
The content is the user couchdb used in the couchdb package.

set name=pkg.fmri value=pkg:/database/$(COMPONENT_NAME)-common@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
set name=pkg.summary value="$(COMPONENT_SUMMARY)"
set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
set name=org.opensolaris.consolidation value=$(CONSOLIDATION)

license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'

group groupname=couchdb gid=56
user username=couchdb ftpuser=false gcos-field="Couchdb Reserved UID" group=couchdb \
password=NP uid=56
-----------------

The name couchdb-common can't be used because it is contained in history.

**Comment by AndWac** [comment]
Am 08.05.21 um 18:30 schrieb grueni:
>
> The content is the user couchdb used in the couchdb package.
>
> set name=pkg.fmri
> value=pkg:/database/$(COMPONENT_NAME)-common@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
> set name=pkg.summary value="$(COMPONENT_SUMMARY)"
> set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
> set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
> set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
>
> license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
>
> group groupname=couchdb gid=56
> user username=couchdb ftpuser=false gcos-field="Couchdb Reserved UID"
> group=couchdb
> password=NP uid=56
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/discussions/6728#discussioncomment-713608>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADET4NY6A7JCIHUYHSSLSXLTMVRJJANCNFSM44MQRYSA>.
>
So, it should be possible to put that into the couchdb-31 package, too.
We don't have anything that could make use of it elsewhere.

**Comment by grueni** [comment]
Ok.
The package can be named couchdb-common3 and be a part of couchdb-31.
The same is the case in package postgresql.95
